File: swagger/models/relationship.py
from .base_model import BaseModel


class Relationship(BaseModel):
	"""
	Jim is a RelationshipName1 of John. John is a RelationshipName2 of
	Jim.

	Attributes:

		id
		The ID of the relationship.

		relationship_name_1
		The name of the first relationship.

		relationship_name_2
		The name of the second relationship.
	"""

	input_map = {
		'Id': 'id',
		'RelationshipName1': 'relationship_name_1',
		'RelationshipName2': 'relationship_name_2',
		}

	output_map = {
		'id': 'Id',
		'relationship_name_1': 'RelationshipName1',
		'relationship_name_2': 'RelationshipName2',
		}

File: swagger/Models/Relationship.php
<?php

namespace App\Mindbody\Models;

/**
 * Jim is a RelationshipName1 of John. John is a RelationshipName2 of
 * Jim.
 */
class Relationship extends BaseModel
{
	/**
	 * The ID of the relationship.
	 * 
	 * @var integer format:int32
	 */
	public $id;
	/**
	 * The name of the first relationship.
	 * 
	 * @var string 
	 */
	public $relationshipName1;
	/**
	 * The name of the second relationship.
	 * 
	 * @var string 
	 */
	public $relationshipName2;

	protected function getInputMap()
	{
		return [
			'Id' => ['id', 'integer', 'int32'],
			'RelationshipName1' => ['relationshipName1', 'string', null],
			'RelationshipName2' => ['relationshipName2', 'string', null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'id' => ['Id', 'integer', 'int32'],
			'relationshipName1' => ['RelationshipName1', 'string', null],
			'relationshipName2' => ['RelationshipName2', 'string', null],
			];
	}
}
